test/notebooks/test_notebooks.py

```diff
@@ -149,7 +149,7 @@ def test_notebooks_advanced(isolated_filesystem, notebook):
     notebook = notebook.replace("examples/tutorials/", "")
     list_name = Path("examples/tutorials/") / notebook
     tested_notebooks.append(str(list_name))
-    res = pm.execute_notebook(notebook, "/dev/null", parameters={"epochs": 1}, timeout=300)
+    res = pm.execute_notebook(notebook, "/dev/null", parameters={"epochs": 1}, timeout=400)
```

Comment:
Q: This is because the notebooks tests sometimes timeout, right?

Reply:
Yes, specifically Encrypted Linear Regression tutorial, and I don't think this timeout is related to change in this PR, just sometimes it exceeded timeout (sometimes not).